backup src/components/Home/Projects.jsx

```diff
@@ -1,3 +1,5 @@
+import { Link } from "lucide-react";
```

Comment:
```suggestion
import Link from "next/link";
```

ig it's the correct import

Reply:
Done.